Add README for Karmada Community Annual Contribution Awards (#183)
karmada-io/community: contributors/award/README.md

### 1. Annual Contributor

Thank you to individual contributors who actively participate and contribute consistently in the community.

- **Quota**: 10 recipients (adjustable based on actual needs)
- **Who**: Individuals who have made important contributions to the Karmada community, including but not limited to code contributions, documentation improvements, community operations, and technical evangelism.

**Recognition and gifts**:
1. Official Karmada community announcement
2. Annual Contributor certificate (electronic)
3. Community swag (the specific form may vary by year)

### 2. Annual Contributing Organization

Thank you to organizations that continue to invest resources and actively support the growth of the Karmada project.

- **Quota**: 10 organizations (adjustable based on actual needs)
- **Who**: Organizations that provide sustained support to the Karmada community, including but not limited to engineering time, technical support, case study sharing, and ecosystem building.

**Recognition**:
1. Official Karmada community announcement
2. Annual Contributing Organization certificate (electronic)

## Eligibility

> The Karmada community recognizes that every contributor and every organization supports the community in different ways, and all of them are valuable.

Reviewer: ```suggestion
Every contribution matters, no matter how big or small. We recognize that people and organizations support Karmada in many different ways, and we value them all.
```

### Individual nominee eligibility

We evaluate outstanding contributions across the dimensions below (meeting any one dimension is sufficient for nomination):

- **Contribution quality**: quality of code implementation, documentation, and community proposals
  - Examples: sound design, feature completeness, documentation accuracy, proposal value, etc.

- **Contribution impact**: positive impact on the community and the cloud-native multi-cluster ecosystem
  - Examples: technical talks, academic papers, community reviews, production adoption, mentoring at community events, etc.

- **Collaboration**: collaborative spirit and ownership demonstrated during community contributions
  - Examples: helping others, responding quickly and resolving issues effectively, providing constructive input in technical discussions, delivering work efficiently, and actively sharing knowledge, etc.

> > Maintainers are core stewards of the project and take key responsibilities for the project's success. To provide more active contributors with recognition and encouragement, the Annual Contribution Awards focus on outstanding contributors outside the Maintainers team.

Reviewer: ```suggestion
### Who can be nominated as an individual contributor?

You don't need to check all the boxes—excelling in any of these areas makes you a great candidate:

**Quality contributions**
- Writing clean, well-designed code
- Creating clear and helpful documentation
- Proposing thoughtful features and improvements

**Community impact**
- Giving talks or writing about Karmada
- Sharing real-world adoption stories
- Mentoring newcomers and helping others learn
- Contributing to the broader cloud-native ecosystem

**Being a great community member**
- Helping others troubleshoot issues
- Jumping into discussions with constructive ideas
- Reviewing PRs and providing helpful feedback
- Sharing knowledge and experience generously

> **Note**: Maintainers are already recognized as core project stewards. To spotlight more contributors, these awards focus on outstanding community members who aren't currently maintainers.
```

### Organization nominee eligibility

We evaluate outstanding organizational contributions across the dimensions below (meeting any one dimension is sufficient for nomination):

- **Organization investment**: resources and engineering time provided to the community
  - Examples: engineering time investment, technical resources support, etc.

- **Organization impact**: positive impact on the community and the cloud-native multi-cluster ecosystem
  - Examples: case study sharing, technical innovation, contributions to ecosystem building, etc.

- **Organization collaboration**: collaboration and shared responsibility with the community

>   - Examples: actively participating in community activities, responding to community needs in a timely manner, providing constructive feedback, etc.

Reviewer: ```suggestion
### What makes an organization a great nominee?

Organizations can make a big difference too. Here's what we look for (again, excelling in any area counts):

**Resource commitment**
- Supporting engineers to contribute their time
- Providing infrastructure, testing environments, or other resources

**Ecosystem impact**
- Sharing production use cases and success stories
- Building tools or integrations that benefit the community
- Driving adoption and innovation

**Active participation**
- Engaging in community discussions and events
- Responding to issues and helping solve problems
- Contributing constructively to the project's direction
```

## Selection Process

The selection is nomination-based. The process is open and transparent: the criteria, process, and results are shared with the community and open to community oversight.

Selection Committee members should remain objective and impartial, and evaluate nominees based on their actual contributions, without being influenced by any external factors.

### Timeline

- **Kickoff**: end of the year
- **Nomination period**: 2–4 weeks
- **Voting period**: 2–4 weeks

> - **Announcement**: announced immediately after the voting period ends

Reviewer: ```suggestion
- **Kickoff**: Early December
- **Nomination period**: Mid-December to mid-January (approximately 4 weeks)
- **Voting period**: 2 weeks
- **Announcement**: Late January

The exact dates will be announced when the selection process begins each year.

> **Note**: The nomination period extends into January to allow time for contribution data to be fully updated and reflected in our systems, ensuring accurate recognition of all contributions made throughout the year.
```

There is no limit on the number of nominees during the nomination period. We welcome and encourage nominations for outstanding contributors and organizations.

**Who can nominate**

- Community members (including Selection Committee members) may nominate any contributor/organization, or self-nominate.

**How to nominate**

Nominate an Annual Contributor or an organization in a comment on the issue. One nominee per comment, in the format:

```
Nominee GitHub ID (or organization name) + nomination reason
```

### Voting

Voting is conducted via an online form by the Selection Committee. Each committee member may cast up to the award quota. The online form is shared within the Selection Committee and is not public.

After the voting period ends, no further votes are accepted. Committee members who did not vote are considered to have abstained.

Exceptional cases:
- **Not enough nominees**: if the number of valid nominees is below the quota, the committee will handle it based on the actual situation.

### Announcement

> Awards will be announced in alphabetical order. Our sincere thanks and congratulations to all award recipients!

Reviewer: ```suggestion
The Selection Committee votes through a private online form. Voting closes at the end of the voting period. Members who don't submit votes are considered abstentions.

**Special situations**:
- **Fewer nominees than slots**: All qualified nominees will receive the award

- **Low participation**: At least half of the committee members must vote for the results to be valid

### Announcement

Award recipients will be announced in alphabetical order. Congratulations to all recipients—and thank you for making Karmada better!

Announcements will include:
- Names of award recipients and their organizations
- Highlights of their contributions
- Publication across official Karmada channels (website, blog, social media)
```

## Selection Process

The selection is nomination-based. The process is open and transparent: the criteria, process, and results are shared with the community and open to community oversight.

Selection Committee members should remain objective and impartial, and evaluate nominees based on their actual contributions, without being influenced by any external factors.

### Timeline

- **Kickoff**: end of the year
- **Nomination period**: 2–4 weeks
- **Voting period**: 2–4 weeks
- **Announcement**: announced immediately after the voting period ends

### Nomination

Nominations are conducted publicly via issues in the [community](https://github.com/karmada-io/community) repository. Separate issues will be created for the two awards, and community members are welcome to participate and provide oversight.

There is no limit on the number of nominees during the nomination period. We welcome and encourage nominations for outstanding contributors and organizations.

**Who can nominate**

- Selection Committee members may nominate or self-nominate

> - Community members who are not on the Selection Committee may self-nominate

Author: Thanks, I think it would be a good idea to seek broader input from community members during the nomination phase.
